src/app/beer_garden/db/mongo/models.py

```diff
@@ -757,3 +757,8 @@ class FileChunk(MongoModel, Document):
     data = StringField(required=True)
     # Delete Rule (2) = CASCADE; This causes this document to be deleted when the owner doc is.
     owner = LazyReferenceField(File, required=False, reverse_delete_rule=CASCADE)
+
+
+class RawFile(Document):
+    name = StringField()
```

Comment:
Weird question, do we actually need a name field?

As of right now it adds no value.